src/dmd/objc_glue.d

```diff
@@ -384,8 +475,21 @@ static:
         return refSymbol;
     }
 
-    Symbol* getMethVarRef(Identifier ident)
+    Symbol* getMethVarRef(const Identifier ident)
     {
         return getMethVarRef(ident.toString());
     }
 }
+
+private:
+
+char[] format(size_t bufLength, Args...)(return ref char[bufLength] buffer,
```

Comment:
Ddoc comment

Reply:
Yes, that's why the "Could use some more documentation" checkbox in the PR description is not checked 😃.